src/test/java/org/jabref/logic/search/query/LuceneQueryParserTest.java

```diff
@@ -1,40 +0,0 @@
-package org.jabref.logic.search.query;
-
-import java.util.stream.Stream;
-
-import org.jabref.model.search.SearchFieldConstants;
-
-import org.apache.lucene.queryparser.classic.ParseException;
-import org.apache.lucene.queryparser.classic.QueryParser;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-
-public class LuceneQueryParserTest {
-
-    public static Stream<Arguments> searchQuires() {
-        return Stream.of(
-                // unicode
-                Arguments.of("any:preissinger", "preißinger"),
-                Arguments.of("any:jesus", "jesús"),
-                Arguments.of("any:breitenbucher", "breitenbücher"),
-
-                // latex
-                Arguments.of("any:preissinger", "\"prei{\\\\ss}inger\""),
-                Arguments.of("any:jesus", "\"jes{\\\\'{u}}s\""),
-                Arguments.of("any:breitenbucher", "\"breitenb{\\\\\\\"{u}}cher\""),
```

Comment:
We should move these tests to somewhere else. Didn't find that in the new code.

Test cases should use `Lübeck` as test. (City name in Germany).

Lübeck, Lubeck, Luebeck, L\"{u}beck

Reply:
> We should move these tests to somewhere else. Didn't find that in the new code.

The bib fields queries are tested in [`src/test/java/org/jabref/logic/search/query/SearchQuerySQLConversionTest.java`](https://github.com/JabRef/jabref/blob/postgresql/src/test/java/org/jabref/logic/search/query/SearchQuerySQLConversionTest.java). I will add these tests there.

> Test cases should use `Lübeck` as test. (City name in Germany).
> 
> Lübeck, Lubeck, Luebeck, L"{u}beck

Only searching with `Lübeck` will work; `Lubeck` and `Luebeck` will not find the entry. This normalization should be done during indexing based on the language. The last term, `L"{u}beck`, will not work because the parser throws an exception due to the `"` character.